stratum/hal/lib/p4/utils.cc

```diff
@@ -117,6 +117,26 @@ std::string ByteStringToP4RuntimeByteString(std::string bytes) {
   return bytes;
 }
 
+::util::Status IsValidMeterConfig(const ::p4::v1::MeterConfig& meter_config) {
+  if (meter_config.cir() > meter_config.pir()) {
```

Comment:
can rate and burst be negative numbers ? according to the spec they are int64

Reply:
Good point:

```json
        {
          "mandatory" : false,
          "read_only" : false,
          "singleton" : {
            "id" : 65547,
            "name" : "$METER_SPEC_PIR_KBPS",
            "repeated" : false,
            "annotations" : [],
            "type" : {
              "type" : "uint64",
              "default_value" : 18446744073709551615
            }
          }
        },
```